cfn-templates/cid-cfn.yml

```diff
@@ -25,7 +25,7 @@ Metadata:
           - DeployComputeOptimizerDashboard
           - CostOptimizationDataCollectionBucketPath
       - Label:
-          default: 'Optional Parameters (do not change unless you know why)'
+          default: 'Optional Parameters. DO NOT CHANGE!'
```

Comment:
DO NOT CHANGE! looks unfriendly, do we really need to change this?

Reply:
Yes. These parameters are not very well tested and error messages can be really confusing. Removing them will require more work, so it is better to put a quick warning until we do so